Net/SMTP.php

```diff
@@ -871,6 +876,95 @@ protected function authPlain($uid, $pwd, $authz = '')
         return true;
     }
 
+        /**
+     * Authenticates the user using the GSSAPI method.
+     *
+     * @note PHP krb5 extension is required and the service principal and
```

Comment:
No such thing as @note in phpdoc, I think. Leave the text but just remove the `@note ` prefix.